implementations/micrometer-registry-datadog/src/main/java/io/micrometer/datadog/DatadogConfig.java

```diff
@@ -72,10 +80,73 @@ default boolean descriptions() {
         return getBoolean(this, "descriptions").orElse(true);
     }
 
+    /**
+     * @return {@code true} if client-side telemetry should be enabled when in Dogstatsd
+     * mode.
+     * https://docs.datadoghq.com/developers/dogstatsd/high_throughput/?code-lang=java#client-side-telemetry
+     */
+    default boolean enableClientSideTelemetry() {
+        return getBoolean(this, "disableClientSideTelemetry").orElse(true);
+    }
+
+    /**
+     * @return The maximum packet size to buffer before sending metrics when using
+     * Dogstatsd mode.
+     * https://docs.datadoghq.com/developers/dogstatsd/high_throughput/?code-lang=java#enable-buffering-on-your-client
+     */
+    default Integer maxPacketSizeBytes() {
+        return getInteger(this, "maxPacketSizeBytes").orElse(-1);
+    }
+
+    /**
+     * @return {@code false} if client-side aggregation should be enabled when in
+     * Dogstatsd mode.
+     * https://docs.datadoghq.com/developers/dogstatsd/high_throughput/?code-lang=java#client-side-aggregation
+     */
+    default boolean enableAggregation() {
+        return getBoolean(this, "enableAggregation").orElse(false);
```

Comment:
Actually, after looking closer, I think `false` may be the better default, since Micrometer is already aggregating and reporting once per step to DogStatsD.